lddwrap/__init__.py

```diff
@@ -162,6 +162,9 @@ def _parse_line(line: str) -> Optional[Dependency]:
             soname=soname, path=dep_path, found=found, mem_address=mem_address)
     else:
         if len(parts) != 2:
+            if line[0] not in {' ', '\t'}:
```

Comment:
I'm very worried that this line might introduce a lot of hard-to-track bugs. I suppose you want to cover the `ldd` output from #14:

```
qt/6.0.2/gcc_64/plugins/sqldrivers/libqsqlpsql.so: /lib/x86_64-linux-gnu/libpq.so.5: no version information available (required by qt/6.0.2/gcc_64/plugins/sqldrivers/libqsqlpsql.so)
```
?

In my opinion, missing leading indention is not an indication enough for missing dependencies.

Why not check for `no version information available` similar how I check for `not found` at line 126?

Reply:
Alrighty, I'm good with this but also remembered a bit along the way.  I guess the 'why not', without claiming it is sufficient to be 'right', is that entering messages one at a time instead of catching all 'message' lines vs. 'dependency' lines generically ends up making more changes.  But sure, this is a undefined formatting and we are (I am?) just extrapolating guesses as to what works out best.  Anyways, it's obviously not that hard to add ignores.

If you are good with my adjustment per your request, then I'm almost good with this.  I'll follow up with one final CI run of my project using this (qt-applications) to make sure it still works as desired in the field.